routemaster/models/database.rb

```diff
@@ -0,0 +1,35 @@
+require 'routemaster/models'
+require 'routemaster/mixins/redis'
+require 'singleton'
+
+module Routemaster
+  module Models
+    # The (Redis) datastore
+    class Database
```

Comment:
Well, in this instance, I think it is. This class allows access to information (metadata) about the (single) database in use. Open to suggestions though!